app/controllers/idv/getting_started_controller.rb

```diff
@@ -0,0 +1,87 @@
+module Idv
+  class GettingStartedController < ApplicationController
+    include IdvStepConcern
+    include StepUtilitiesConcern
+
+    before_action :confirm_agreement_needed
+
+    def show
+      analytics.idv_doc_auth_getting_started_visited(**analytics_arguments)
+
+      # Register both Welcome and Agreement steps in DocAuthLog
+      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+        call('welcome', :view, true)
+      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+        call('agreement', :view, true)
+
+      @sp_name = decorated_session.sp_name || t('doc_auth.info.no_sp_name')
+      @title = t(
+        'doc_auth.headings.getting_started',
+        sp_name: decorated_session.sp_name || t('doc_auth.info.no_sp_name').downcase,
```

Comment:
calling `#downcase` on the output of `I18.t` is an anti-pattern IMO. In English this might have the desired affect, but that's not consistent across languages, I would recommend duplicating the string or not downcasing if possible

Reply:
Agreed! Fixed by removing this translation tag altogether and replacing with APP_NAME.